1-D_Array/C/odd_even.c

```diff
@@ -0,0 +1,36 @@
+//program to find odd and even numbers from a array
+
+#include <stdio.h>
+
+void main()
+{
+    int a[20],i,n;
+    printf("Enter the array size ");
+    scanf("%d",&n);
+    printf("Enter the array elements\n");
+    for(i=0;i<n;i++){
```

Comment:
Add space after ``for`` and after semicolons in all places.
Also around all operators.